src/index.js

```diff
@@ -320,6 +320,12 @@ export function initialize(clientSdkKey, flagKeys, user, specifiedOptions, platf
           console.warn("[UL] Disabling local storage in production is not recommended. Please enable it. For more information see: https://docs.unlaunch.io/docs/sdks/javascript-library#client-configuration")
       }
 
+      if(clientSdkKey.split('-')[1] !== 'public'){
+        console.warn("You're using NOT using public key to connect to Unlaunch. If your application is client-side, not using public key can compromise security. To obtain the public SDK key, please sign in the Unlaunch Console at https://app.unlaunch.io" 
```

Comment:
We detected that you're not using the public SDK key to connect to Unlaunch from a browser SDK. For security reasons, we highly recommend using public key to access Unlaunch from JavaScript or other public SDKs. To obtain the public SDK key, please sign in the Unlaunch Console at https://app.unlaunch.io" 
                      +". Then on the right sidebar, click on 'Settings'. Then from the 'Projects' tab. Copy the 'Browser / Public Key' for the " 
                      +"environment you want to connect to, and provide it to this SDK. For more information, visit: https://docs.unlaunch.io/docs/sdks/sdk-keys"